### DISCUSSION TITLE
Transactions Tracing not showing up with Hardhat

### DISCUSSION THREAD

**Discussion by ebnth-zorea** [tryethernal/ethernal#71] [Q&A] [answered]
- Enabled the option in settings for "Trace on a Hardhat network", which is active. 
- Updated and restarted hardhat node and browser
- Deploy with hardhat script using Ethernal
```
 await hre.ethernal.push({
      name: 'Template',
      address: templ.address
  })
```
- Make transactions with data payload
- Transactions are indexed by Ethernal
- No trace section shown in any

What am I missing?

**Comment by antoinedc** [comment, marked as answer]
Hi,

 In the hardhat logs, do you have a line that says `[Ethernal] Tracing transaction...`, around the tx log?

**Reply by ebnth-zorea** [reply]
Yes, confirmed this entry appears when sending a TX on hardhat.
It's the UI that has no "Trace" section as in the tutorial video.

My project Hardhat version is: 2.6.4
Ethernal version is: 0.7.6

**Reply by antoinedc** [reply]
Ok, thanks! Can you send me by mp on Discord (@antoinedc) the email you signed up with, please?
